Gemfile.lock

```diff
@@ -1,105 +1,104 @@
 PATH
```

Comment:
we should probably [remove Gemfile.lock from the repo](https://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/)